docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>I suppose that it would be better to include an observation table and an hdu index table in the gammapy-data repository for clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done in https://github.com/gammapy/gammapy-data/pull/22/files